docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx

```diff
@@ -45,16 +45,76 @@ import '@aws-amplify/ui-react/styles.css';
 ```
 </ExampleCode>
 
-At a minimum you must include the `accessLevel` and `maxFileCount` props. `accessLevel` refers to the [Amplify Storage access level](https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js/), which is `'guest' | 'protected' | 'private'`.
+At a minimum you must include the `path` and `maxFileCount` props. `path` refers to the S3 image path that will be prefixed to each file key. It is either a `string` or a callback function that accepts the current user's Cognito `identityId` and returns a `string`. See [upload files](https://docs.amplify.aws/react/build-a-backend/storage/upload-files/) 
+
+<Alert variation="info" heading="Version 3.0.18">
+  Using `@aws-amplify/ui-react-storage` version 3.0.18 or below and looking for docs on `accessLevel`? See [`accessLevel`](#accesslevel-props)
+</Alert>
 
 <Example>
-  <DefaultStorageManagerExample />
+  <DefaultExample />
   <ExampleCode>
-    ```jsx file=./examples/DefaultStorageManagerExample.tsx
+    ```jsx file=./examples/Default.tsx
     ```
   </ExampleCode>
 </Example>
 
+### Private or Protected Buckets
+When uploading to private or protected S3 buckets, you'll need to wrap your app in the `Authenticator`, allowing the `StorageManager` component to infer the Cognito `identityId` of the currently signed-in user. This can be done directly with the `Authenticator` component or with `withAuthenticator`, as shown in [Add the Authenticator](/connected-components/authenticator#step-3-add-the-authenticator).
+<Example>
```

Comment:
Seems like this needs a description to following the blurb about the Authenticator:

```suggestion

The example below shows configuring the StorageManager to upload to the `protected` folder under the users' identity id.

<Example>
```